docker/wait-for-it.sh

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env bash
```

Comment:
It's fine this. But I normally use `apt install wait-for-it` in the Dockerfile and then you can use it in the docker-compose and you don't need to have this script in the repo.